tinygrad/renderer/assembly.py

```diff
@@ -177,13 +177,15 @@ def _cast(a, dtype:DType, atype:DType, bitcast=False, u=None, pred=False):
       if uop is UOps.IF:
         assert src[0].dtype is not None
         kk(*self.render_bra(f"IF_{r[src[0]][1:]}_{uops.index(u)}", _cast(r[src[0]], dtypes.bool, src[0].dtype, u=u, pred=True)))
+        kk(f"bra ENDIF_{r[src[0]][1:]}_{uops.index(u)};")
```

Comment:
why doesn't this use render_bra like above?

Reply:
Will swap that over